git/README.md

```diff
@@ -161,28 +155,44 @@ git branch --delete <branch-name>
 
 ## Pull Request Template
 
-Please add a [Pull Request Template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) to your repo, to ensure that all pull requests follow our guide:
+Please add a [Pull Request Template] to your repo,
+to ensure that all pull requests follow our guide:
 
-```
-## Overview
+See our [Standard Pull Request Template](./commit_template.txt) in this directory.
 
-Please include a summary of the change and which issue is fixed.
-Please also include relevant motivation and context.
-List any dependencies that are required for this change.
+## Write Useful Commit Messages
 
-## Screenshots
+Please ensure that you follow [commit message] [best practices]:
 
-Visual changes should include a screenshot.
+1. Separate subject from body with a blank line
+1. Limit the subject line to 50 characters
+1. Capitalize the subject line
+1. Do not end the subject line with a period
+1. Use the imperative mood in the subject line
```

Comment:
Is it just me? I had to double-check "the imperative mood"

Reply:
I had to check that as well. Maybe an example would be good?

e.g. "Fix the bug" instead of "Fixed the bug"

Or maybe a link to a reference? e.g. https://www.thoughtco.com/imperative-mood-grammar-1691151

Reply:
Also, does this line mean to capitalize the first word, or write the whole subject in title case? I'd guess the first, but wanted to check.

 1. Capitalize the subject line

Reply:
> Also, does this line mean to capitalize the first word, or write the whole subject in title case? I'd guess the first, but wanted to check.
> 
> 1. Capitalize the subject line

I thought that was clear, but maybe "Capitalize the first word of the subject line"

Reply:
Yeah, I might be overthinking it. That's how I would read it as well.

The CSS `text-transform` property has confused me about this, since there `capitalize` means capitalize every word.

Reply:
Great feedback! I made these changes.